### PR TITLE
Automate npm-publish using github action and minor improvements

### PR DESCRIPTION
**Pull request by KapilTR**
- Added npm-publish.yml file to automate npm package publish upon making github release.
- prompt for set-provider command
- version change in tezster.js and package.json to automaticaly publishing package

### CODE REVIEW

File: tezster-manager.js [hunk 1]
```diff
@@ -34,8 +34,8 @@ class TezsterManager{
         this.setup.getLogFiles();
     }
 
-    setProvider(){
-        this.accounts.setProvider(args);
+    setProvider(accountSetProviderParameterValues){
```
Comment (jvtezsure) [inline tezster-manager.js:37]: why are you combinign multiple items in a single CL? 
it should be done in a separate PR.

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+
+  publish-npm:
```
Comment (jvtezsure) [inline .github/workflows/npm-publish.yml:17]: How would you handle change of version in tezster.js?

File: tezster-manager.js (hunk 1 above)
Comment (KapilTR) [reply tezster-manager.js:37]: I will take care of this further.

File: .github/workflows/npm-publish.yml (hunk 2 above)
Comment (KapilTR) [reply .github/workflows/npm-publish.yml:17]: That will auto fetch for tezster version from package.json file. And have to merge PR into master after changing versions in tezster.js and package.json both files.

File: modules/setup/setup.js [hunk 3]
```diff
@@ -25,6 +25,11 @@ class Setup {
 
             docker.getContainer(CONTAINER_NAME).inspect().then((result) => {
                 if(result.State.Status === 'exited' || result.State.Status === 'running') {
+                    if(result.Config.Image !== IMAGE_TAG) {
+                        Helper.errorLogHandler(`Container was up with previous image tag....`,
```
Comment (jvtezsure) [inline modules/setup/setup.js]: Will this error be shown to user? 
why can't we handle without confusing end user about image version and all?

**Review by jvtezsure — CHANGES_REQUESTED** [review]

**Review by jvtezsure — APPROVED** [review]
